index.html

```diff
@@ -17,7 +17,7 @@
         </a>
       </nav>
     </header>
-    <header class="header-box">
```

Comment:
- [ ] As requested by the previous reviewer, the navbar includes the 4 option in the **desktop** version, and adjust the spacing between them.
 
- Figma design 
![Screenshot from 2022-03-27 02-12-29](https://user-images.githubusercontent.com/26767383/160256793-3f129545-ce05-45d3-a2a5-3d2abf83ba71.png)